Add option to not add X-Forwarded headers (#417)
trinodb/trino-gateway: docs/installation.md

or more Trino clusters. The following Trino configuration tips should be 
taken into account for all clusters behind the Trino Gateway.

> If all client/server communication is to be routed through the Trino Gateway, 

Reviewer: ```suggestion
If all client/server communication is routed through the Trino Gateway, 
```

to Trino correctly. However, the URL for subsequent next URIs for more results
in a query provided by Trino is then using the local URL of the Trino cluster,
and not the URL of the Trino Gateway. This circumvents the Trino Gateway for all
these requests. In scenarios, where the local URL of the Trino cluster is private 

> to the Trino cluster on the DNS/network level, these following calls might not work

Reviewer: ```suggestion
to the Trino cluster on the network level, these following calls do not work
```
